packages/metal-dom/src/domNamed.js

```diff
@@ -209,15 +209,11 @@ export function buildFragment(htmlString) {
  * Checks if the first element contains the second one.
  * @param {!Element} element1
  * @param {!Element} element2
+ * @deprecated Use element1.contains(element2) directly instead of this method
```

Comment:
Can you move this on top of the other `@param` lines? 🙏

Reply:
Done! :)